### PR TITLE
Update api-status-codes.md

### PR DESCRIPTION
**Pull request by bc-tgomez**
Updated API Status 400 to include url restriction.

# [DEVDOCS-3358](https://jira.bigcommerce.com/browse/DEVDOCS-3358)

## Updated API Status Codes
* Included URL restriction return request msg for status code 400. 
* Cleaned up and made small editorial edits.

### CODE REVIEW

File: docs/api-docs/getting-started/api-status-codes.md [hunk 1]
```diff
@@ -39,15 +39,16 @@ The BigCommerce API responds to requests with different HTTP status codes depend
 
 | Code | Text | Purpose |
 |-|-|-|
-| **400** | **Bad Request** | Issued when a malformed request was sent.
+| **400** | **Bad Request** | Issued when a malformed request was sent. |
+||| The request can not be completed due to a URL restriction. |
```
Comment (bc-andreadao) [inline docs/api-docs/getting-started/api-status-codes.md:43]:
```suggestion
||| The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |

```

File: docs/api-docs/getting-started/api-status-codes.md [hunk 2]
```diff
@@ -39,15 +39,17 @@ The BigCommerce API responds to requests with different HTTP status codes depend
 
 | Code | Text | Purpose |
 |-|-|-|
-| **400** | **Bad Request** | Issued when a malformed request was sent.
+| **400** | **Bad Request** | Issued when a malformed request was sent. |
+||| The request can not be completed due to a URL restriction. Check the URL for ports that may conflict with site permissions. |
```
Comment (markcmurphy) [inline docs/api-docs/getting-started/api-status-codes.md:43]: When I preview this doc, there's something off about the table formatting. Can you take a look again?

Comment (bc-tgomez) [reply docs/api-docs/getting-started/api-status-codes.md:43]: Definitely, I'll take a look at this.

Comment (bc-tgomez) [reply docs/api-docs/getting-started/api-status-codes.md]: Fixed? Maybe? Preview looks better.

File: docs/api-docs/getting-started/api-status-codes.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # API Status Codes
 
 
-The BigCommerce API responds to requests with different HTTP status codes depending on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.
+The BigCommerce API responds to a request with different HTTP status codes depending on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.
```
Comment (slsriehl) [inline docs/api-docs/getting-started/api-status-codes.md:4]:
```suggestion
The BigCommerce API responds to each request with an HTTP status code that depends on the result from the request. Error responses might also include an error message in the body to assist in resolving the problem.

```

Comment (slsriehl) [reply docs/api-docs/getting-started/api-status-codes.md:4]: @bc-tgomez -- I like the shift to singular request.  How do you feel about this?

**Review by slsriehl — APPROVED** [review]
hear, hear. 👏

**Pull request merged**